### PR TITLE
Add dependencies in Java11-specific libraries

### PR DESCRIPTION
**Pull request by cipherboy**
This ensures lib/ gets created prior to any Java11-specific symlinks.

`Signed-off-by: Alexander Scheel <ascheel@redhat.com>`

### CODE REVIEW

**Review by edewata — APPROVED** [review]

File: base/server/CMakeLists.txt [hunk 1]
```diff
@@ -118,6 +118,7 @@ add_custom_command(
 
 if(JAVA_VERSION GREATER 10)
     add_custom_target(pki-server-java11plus-lib ALL
```
Comment (cipherboy) [inline base/server/CMakeLists.txt:121]: @edewata This target never executes. In particular, the links `common/lib/jaxb-api.jar` don't ever seem to get created according to the [build log](https://pipelines.actions.githubusercontent.com/bmG54JoT9T4XjcmeUGdblFiDgrq6NGpbGGR1QtrV6gjRWh3H1P/_apis/pipelines/1/runs/960/signedlogcontent/5?urlExpires=2020-09-16T15%3A16%3A37.2496997Z&urlSigningMethod=HMACV1&urlSignature=GiolloZk0L7Ls0kIySfuNJoWgQZOJR09FiNY1Y8cpCA%3D).

`JAVA_VERSION` gets set to 11.

Comment (cipherboy) [reply base/server/CMakeLists.txt:120]: Turns out it was because we used `%{java_version}` in our spec instead of `${java_version}`. Ah well :-)

**Comment by cipherboy** [comment]
@edewata This passes with CI now. I'll rebase (to remove stuff from #528) and then merge if that's alright?

**Comment by edewata** [comment]
Yes, go ahead. Thanks!

**Comment by abbra** [comment]
Can we have a release in Fedora 33/rawhide, please?

Right now we cannot deploy anything with Rawhide/F33 due to this bug.

**Comment by cipherboy** [comment]
https://koji.fedoraproject.org/koji/taskinfo?taskID=51738402
https://koji.fedoraproject.org/koji/taskinfo?taskID=51738802

Will update bodhi once built.

**Pull request merged**